module.py

```diff
@@ -97,4 +97,5 @@ def shutdown(self) -> None:
             return
 
     def notify(self, notify_type: str, notify_id: str) -> None:
-        self.log.debug(f"recv notify {notify_type}")
+        # self.log.debug(f"recv notify {notify_type}")
```

Comment:
just drop the line, no need to comment it. I'd leave the function with the `pass` so we can use it in the near future.